2019/12/18 15:15:49.123 - MainThread - AndorFunctions.py.__init__.48 - ERROR
Andor functions dll file not found.

Traceback (most recent call last):
  File "./andorcamera\AndorFunctions.py", line 46, in __init__
    self.dll = cdll.LoadLibrary(dllpath) # note dll path must be absolute, not relative
  File "C:\Users\qgtx64\.conda\envs\pydexenv\lib\ctypes\__init__.py", line 442, in LoadLibrary
    return self._dlltype(name)
  File "C:\Users\qgtx64\.conda\envs\pydexenv\lib\ctypes\__init__.py", line 364, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: [WinError 126] The specified module could not be found
2019/12/18 15:15:49.489 - MainThread - cameraHandler.py.__init__.64 - WARNING
Andor EMCCD not initialised.
'Andor' object has no attribute 'dll'


2019/12/18 15:15:49.534 - MainThread - imsaver.py.__init__.73 - WARNING
Could not create image storage path. Using current directory instead
[WinError 5] Access is denied: 'C:\\Users\\Lab'


2019/12/18 15:16:57.623 - Dummy-1 - networker.py.run.114 - ERROR
Python server: client terminated connection before send. Resending message.
[WinError 10053] An established connection was aborted by the software in your host machine


2019/12/18 15:16:57.627 - Dummy-1 - networker.py.run.122 - ERROR
Python server: client terminated connection before receive.
[WinError 10053] An established connection was aborted by the software in your host machine


2019/12/18 15:18:03.014 - Dummy-1 - networker.py.run.114 - ERROR
Python server: client terminated connection before send. Resending message.
[WinError 10053] An established connection was aborted by the software in your host machine


2019/12/18 15:18:03.018 - Dummy-1 - networker.py.run.122 - ERROR
Python server: client terminated connection before receive.
[WinError 10053] An established connection was aborted by the software in your host machine


2019/12/18 15:20:14.211 - MainThread - master.py.closeEvent.340 - WARNING
camera safe shutdown failed.
'Andor' object has no attribute 'dll'


